<?xml version="1.0"?>
<!--***********************************************************
	Joint State Monitor Node
	Emily Sheetz, NSTGRO VTE 2023
************************************************************-->
<launch>
	
	<!-- parameters for specifying joint state topic -->
	<arg name="joint_state_topic" default="/ihmc_ros/valkyrie/output/joint_states"/>

	<node pkg="val_soft_estop_monitor" type="joint_state_monitor_node" name="ValkyrieJointStateMonitorNode" output="screen">
		<param name="joint_state_topic" type="string" value="$(arg joint_state_topic)"/>
		<!--<param name="" type="" value=""/> -->
	</node>

	<!-- rosrun rqt_reconfigure rqt_reconfigure -->
	<node pkg="rqt_reconfigure" name="rqt_reconfigure_joint_state_monitor" type="rqt_reconfigure" output="screen"/>

</launch>
